ui-components/src/actions/Button.docs.mdx

```diff
@@ -50,12 +50,12 @@ You can apply CSS variables to the `.button` selector to customize the appearanc
 | `--alert-appearance-hover-background-color`   | <Swatch colorVar="--bloom-color-alert-dark" />   | Hover state alert background color   |
 | `--alert-appearance-hover-border-color`       | <Swatch colorVar="--bloom-color-alert" />        | Hover state alert border color       |
 | `--alert-appearance-hover-label-color`        | <Swatch colorVar="--bloom-color-white" />        | Hover state alert label color        |
-| `--warn-appearance-background-color`          | <Swatch colorVar="--bloom-color-warn" />         | Warning background color             |
-| `--warn-appearance-border-color`              | <Swatch colorVar="--bloom-color-warn" />         | Warning border color                 |
-| `--warn-appearance-label-color`               | <Swatch colorVar="--bloom-color-white" />        | Warning label color                  |
-| `--warn-appearance-hover-background-color`    | <Swatch colorVar="--bloom-color-warn-dark" />    | Hover state warning background color |
-| `--warn-appearance-hover-border-color`        | <Swatch colorVar="--bloom-color-warn" />         | Hover state warning border color     |
-| `--warn-appearance-hover-label-color`         | <Swatch colorVar="--bloom-color-white" />        | Hover state warning label color      |
+| `--warning-appearance-background-color`       | <Swatch colorVar="--bloom-color-warn" />         | Warning background color             |
```

Comment:
I'm sure I'm just confused - but shouldn't the variables in this doc file be only the variables in the {component}.scss file? I don't see the ---warning-*  variables in Button.scss file.

Reply:
So, the Button and Tag components (and possibly others in the future) inherit a collection of appearance styles (Primary, Warning, Alert, etc.). But you're right, this isn't specific just to buttons, and I'd love to move it out of the Button story per se. Maybe if we add a generic docs page to Storybook with "global" styling examples, we can relocate this over there.

Reply:
That makes sense!